site/content/docs/4.3/components/offcanvas.md

```diff
@@ -0,0 +1,217 @@
+    <button type="button" class="close text-reset" data-dismiss="offcanvas" aria-label="Close">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+</div>
+{{< /example >}}
+
+## Color schemes
+
+Theming offcanvas is very easy by using our [color utilities]({{< docsref "/utilities/colors" >}}).
+Any combination of `bg-*` and `text-{color}` utilities, can give a different result
+
+{{< example >}}
+<p>
+  <button class="btn btn-primary" type="button" data-toggle="offcanvas" data-target="#offcanvasExample31" aria-expanded="false" aria-controls="offcanvasExample31">Colored Offcanvas</button>
+</p>
+<div class="offcanvas bg-primary text-white" data-body="scroll" tabindex="-1" id="offcanvasExample31">
+  <div class="offcanvas-header">
+    <div class="h4 col text-center">Hey I got New Colors</div>
+  </div>
+</div>
+
+{{< /example >}}
+
+## Body Options
+
+By default, when an offcanvas is visible, body cannot be scrolled. You can use the following data-options:
+
+- `data-body="scroll"` enables scrollBar on body when offcanvas is open
+- `data-body="backdrop"` hides scrollBar and creates a backdrop on body, when offcanvas is open
+
+{{< example >}}
+<p>
+  <button class="btn btn-primary" type="button" data-toggle="offcanvas" data-target="#offcanvasExample4" aria-expanded="false" aria-controls="offcanvasExample4">Enable body scrolling </button>
+  <button class="btn btn-primary" type="button" data-toggle="offcanvas" data-target="#offcanvasExample5" aria-expanded="false" aria-controls="offcanvasExample5">Body backdrop</button>
+</p>
+<div class="offcanvas bg-dark text-white" data-body="scroll" tabindex="-1" id="offcanvasExample4">
+  <div class="offcanvas-header">
+    <div class="h4 col text-center">Wow, body scroller is with us again</div>
+  </div>
+</div>
+<div class="offcanvas bg-dark text-white" data-body="backdrop" tabindex="-1" id="offcanvasExample5">
+  <div class="offcanvas-header">
+    <div class="h4 col text-center">Body is covered by a backdrop</div>
+  </div>
+</div>
+{{< /example >}}
+
+## Accessibility
+
+Make sure to add `aria-expanded` to the control element. This attribute explicitly conveys the current state of the collapsible element tied to the control to screen readers and similar assistive technologies. If the collapsible element is closed by default, the attribute on the control element should have a value of `aria-expanded="false"`. If you've set the collapsible element to be open by default using the `show` class, set `aria-expanded="true"` on the control instead. The plugin will automatically toggle this attribute on the control based on whether or not the collapsible element has been opened or closed (via JavaScript, or because the user triggered another control element also tied to the same collapsible element). If the control element's HTML element is not a button (e.g., an `<a>` or `<div>`), the attribute `role="button"` should be added to the element.
```

Comment:
Do we need to update the language here to be changed from `collapsible` to `offcanvas`? Or, to avoid duplication, should we point folks to this section in the collapse docs?

Reply:
I chose the 'link' way, to avoid extra text.